scripts/seed.py

```diff
@@ -76,6 +92,13 @@ def create_challenges(number_of_challenges=3, host_team=None):
 
 
 def create_challenge(title, start_date, end_date, host_team):
+    """
+    Creates a challenge.
+
+    Add a challenge with the parameters title, short_description, description, terms_and_condition,
```

Comment:
This is redundant. Code is self explanatory. The only need to add comment was just to give an overview of what a function is doing. I think this should not be there @deshraj